File: Logic_HandsOn/Java/Problems on Array,Pattern etc/Program1.java
////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//	Program1:
//		a program which accept one number from user and print that  number of even numbers on screen.
//
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

import java.util.Scanner;

class Solution
{
	
	void PrintEven(int iNo)
	{
		if(iNo<0)
		{
			return;
		}

		int iCnt=0;
		for(int olc=1;olc<=100;olc++)
		{
			if(olc%2==0)
			{
				System.out.print(olc+" ");
				iCnt++;
			}
			if(iCnt==iNo)
			{
				break;
			}
		}
		System.out.println();
	}
}

class Program1
{
	public  static void main(String[] args)
	{
		Scanner s = new Scanner(System.in);
		int iNo = s.nextInt();

		Solution sObj = new Solution();
		sObj.PrintEven(iNo);
	}
}

/*
OUTPUT:

8
2 4 6 8 10 12 14 16

*/

File: Logic_HandsOn/C++/Problems on Digits,String,Array,Pointers etc/program1.cpp
////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//	Program1:
//		a program which accept one number from user and print that  number of even numbers on screen.
//
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

#include<iostream>
using namespace std;

class Solution
{
	public:
	void PrintEven(int iNo)
	{

		if(iNo<=0)
		{
	
			return;
		}

		int iCnt=0;
		for(int olc=1;olc<=100;olc++)
		{
			if(olc%2==0)
			{
				cout<<olc<<" ";
				iCnt++;	
			}
			if(iCnt==iNo)
			{
				return;
			}
		}	
	}
};	

int main()
{

	int iValue=0;
	cout<<"Enter number\n";
	cin>>iValue;
	
	Solution *sObj = new Solution();
	sObj->PrintEven(iValue);

	return 0;
}

/*
OUTPUT:

Enter number
7
2 4 6 8 10 12 14

*/
